stack.yaml

```diff
@@ -7,8 +7,6 @@ packages:
   - .
 
 extra-deps:
-  - lorentz-0.6.0@sha256:546c75525aa41a3e5782aaa4da17bd13b72a57ecee9836c553582b675bb5f4f1,4095
-  - morley-1.6.0@sha256:62f2e31fa68c4e67fb4f4626664b5a0f02185c3d48f106b5dd10fb7efe569922,7419
```

Comment:
I am pretty sure you can still take this packages from Hackage rather than git repos. Just update them to 0.6.1 and 1.7.0